services/brig/src/Brig/Team/Util.hs

```diff
@@ -11,19 +11,34 @@ import Control.Error
 import qualified Brig.IO.Intra as Intra
 import qualified Data.Set as Set
 
-data TeamOwnershipStatus = IsOnlyTeamOwner | IsOneOfManyTeamOwners | IsNotTeamOwner | NoTeamOwnersAreLeft
+-- | Every team must have at least one owner with an email address for billing and
+-- administration.  'TeamOwnershipStatus' distinguishes all the relevant cases.
+data TeamOwnershipStatus
+  = IsOnlyTeamOwnerWithEmail
+  | IsOneOfManyTeamOwnersWithEmail
+  | IsTeamOwnerWithoutEmail
+  | IsNotTeamOwner
   deriving (Eq, Show, Bounded, Enum)
 
 -- | A team owner is a team member with full permissions *and* an email address.
```

Comment:
Can you adjust this comment then please?

Reply:
oops, thanks.  i removed the comment, as it doesn't really belong here.